plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSystemTables.java

```diff
@@ -103,4 +106,175 @@ public void testPropertiesTable()
             assertUpdate("DROP TABLE IF EXISTS " + tableName);
         }
     }
+
+    @Test
+    public void testPartitionsTable()
```

Comment:
Please add another test which has uppercase characters in partitioned column names. You will need to register the table from resources.

Reply:
Added `TestDeltaLakeSystemTables#testPartitionsTableCaseSensitiveColumns`.